### ISSUE TITLE
Fatal error deleting multiple posts

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-loader#19]
I got a Fatal error from oik-unloader when attempting to Move to trash 3 block posts. 
The Fatal was followed by a couple of Notices, a Warning then another Fatal. 

URL was https://blocks.wp.a2z/wp-admin/edit.php?orderby=date&order=desc&s=&post_status=all&post_type=block&_wpnonce=3933e1c951&_wp_http_referer=%2Fwp-admin%2Fedit.php%3Fpost_type%3Dblock%26orderby%3Ddate%26order%3Ddesc&action=trash&m=0&seo_filter=&readability_filter=&paged=1&post%5B%5D=10919&post%5B%5D=10918&post%5B%5D=10917&action2=trash

Fatal error: Uncaught TypeError: Illegal offset type in isset or empty in C:\apache\htdocs\wp-a2z\wp-content\mu-plugins\oik-loader-mu.php:181 
```
Stack trace: 
#0 C:\apache\htdocs\wp-a2z\wp-content\mu-plugins\oik-loader-mu.php(66): oik_loader_mu_query_plugins(Array, Array) 
#1 C:\apache\htdocs\wp-a2z\wp-content\mu-plugins\oik-loader-mu.php(39): oik_loader_try_file('oik-loader') 
#2 C:\apache\htdocs\wp-a2z\wp-content\mu-plugins\oik-loader-mu.php(30): oik_loader_mu_loaded() 
#3 C:\apache\htdocs\wp-a2z\wp-settings.php(353): include_once('C:\\apache\\htdoc...') 
#4 C:\apache\htdocs\wp-a2z\wp-config.php(126): require_once('C:\\apache\\htdoc...') 
#5 C:\apache\htdocs\wp-a2z\wp-load.php(50): require_once('C:\\apache\\htdoc...') 
#6 C:\apache\htdocs\wp-a2z\wp-admin\admin.php(34): require_once('C:\\apache\\htdoc...') 
#7 C:\apache\htdocs\wp-a2z\wp-admin\edit.php(10): require_once('C:\\apache\\htdoc...') 
#8 {main} thrown in C:\apache\htdocs\wp-a2z\wp-content\mu-plugins\oik-loader-mu.php on line 181
```

**Comment by bobbingwide** [comment]
### Workaround 
Trash the items one at a time.

**Comment by bobbingwide** [comment]
I got a similar problem when using bulk update > Activate for all the ACF groups in s.b/oikcom

https://s.b/oikcom/wp-admin/edit.php?s=&post_status=all&post_type=acf-field-group&_wpnonce=db1ddb30a2&_wp_http_referer=%2Foikcom%2Fwp-admin%2Fedit.php%3Fpost_type%3Dacf-field-group&action=acfactivate&m=0&paged=1&post%5B%5D=49143&post%5B%5D=49002&post%5B%5D=49092&post%5B%5D=49217&post%5B%5D=49226&post%5B%5D=49023&post%5B%5D=49243&post%5B%5D=49115&post%5B%5D=49163&post%5B%5D=49102&post%5B%5D=49260&action2=acfactivate

```
<b>Fatal error</b>:  Uncaught TypeError: Illegal offset type in isset or empty in C:\apache\htdocs\oikcom\wp-content\mu-plugins\oik-loader-mu.php:137
--
  | Stack trace:
  | #0 C:\apache\htdocs\oikcom\wp-content\mu-plugins\oik-loader-mu.php(43): oik_loader_mu_query_plugins(Array, Array)
  | #1 C:\apache\htdocs\oikcom\wp-content\mu-plugins\oik-loader-mu.php(30): oik_loader_mu_loaded()
  | #2 C:\apache\htdocs\oikcom\wp-settings.php(378): include_once('C:\\apache\\htdoc...')
  | #3 C:\apache\htdocs\oikcom\wp-config.php(89): require_once('C:\\apache\\htdoc...')
  | #4 C:\apache\htdocs\oikcom\wp-load.php(50): require_once('C:\\apache\\htdoc...')
  | #5 C:\apache\htdocs\oikcom\wp-admin\admin.php(34): require_once('C:\\apache\\htdoc...')
  | #6 C:\apache\htdocs\oikcom\wp-admin\edit.php(10): require_once('C:\\apache\\htdoc...')
  | #7 {main}
  | thrown in <b>C:\apache\htdocs\oikcom\wp-content\mu-plugins\oik-loader-mu.php</b> on line <b>137</b><br />
 ```

**Comment by bobbingwide** [comment]
This is now fixed in oik-loader v1.4.1

**Comment by bobbingwide** [comment]
Retested with v1.4.2 on cwiccer.com, deleting two unwanted posts.
